### PR TITLE
feat(js): hide branding

### PR DESCRIPTION
**Pull request by BiswaViraj**
### Screenshots

image

<details>
<summary><strong>Expand for optional sections</strong></summary>

### Special notes for your reviewer

</details>

### CODE REVIEW

File: packages/js/src/ui/context/InboxContext.tsx [hunk 1]
```diff
@@ -42,6 +44,7 @@ export const InboxProvider = (props: InboxProviderProps) => {
     ...STATUS_TO_FILTER[NotificationStatus.UNREAD_READ],
     tags: props.tabs.length > 0 ? props.tabs[0].value : [],
   });
+  const [hideBranding, setHideBranding] = createSignal(false);
```
Comment (BiswaViraj) [inline packages/js/src/ui/context/InboxContext.tsx]: ❓ **question:** ‏Added it to the inboxContext, let me know if i should move it to a different context or a new one

File: packages/js/src/ui/context/InboxContext.tsx [hunk 2]
```diff
@@ -88,6 +91,17 @@ export const InboxProvider = (props: InboxProviderProps) => {
     setFilter((old) => ({ ...old, tags: firstTab?.value ?? [] }));
   });
 
+  useNovuEvent({
+    event: 'session.initialize.resolved',
+    eventHandler: ({ data }) => {
```
Comment (SokratisVidros) [inline packages/js/src/ui/context/InboxContext.tsx]: Do we have strong types for this?

File: packages/js/src/ui/context/InboxContext.tsx (hunk 1 above)
Comment (SokratisVidros) [reply packages/js/src/ui/context/InboxContext.tsx]: I think this is the right place for now.

**Review by SokratisVidros — APPROVED** [review]

File: packages/js/src/ui/context/InboxContext.tsx (hunk 2 above)
Comment (BiswaViraj) [reply packages/js/src/ui/context/InboxContext.tsx]: Yes, it auto infers the correct type on the basis of the event name

**Pull request closed without merging**